lib/asciidoctor/js/postscript.rb

```diff
@@ -1,4 +1,3 @@
-require 'asciidoctor/converter'
```

Comment:
That seems strange because it's required in 1.5.5. See https://github.com/asciidoctor/asciidoctor/blob/v1.5.5/lib/asciidoctor.rb#L1611

In other words, there's no difference between 1.5.5 and 1.5.6.dev in this case, so I don't understand why there would be a difference in Asciidoctor.js.